litestar/_openapi/parameters.py

```diff
@@ -206,13 +206,13 @@ def create_parameter_for_handler(
     dependency_providers = route_handler.resolve_dependencies()
     layered_parameters = route_handler.resolve_layered_parameters()
 
-    unique_handler_fields = tuple(
+    unique_handler_fields = (
```

Comment:
i know unrelated, but couldn't help myself - we only iterate over these so no need to construct the tuple.